content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -209,7 +209,7 @@ would have failed due to conflicting ownership.
 
 The merging strategy, implemented with Server Side Apply, provides a generally
 more stable object lifecycle. Server Side Apply tries to merge fields based on
-the fact who manages them instead of overruling just based on values. This way
+which objects manage them instead of overruling based on values. This way
```

Comment:
@apelisse .
Would you have time to clarify this sentence? I'd like to replace "the fact who manges" and "just based":

Server Side Apply tries to merge fields based on
the objects managing the fields instead of overruling fields based only on values.
or
Server Side Apply tries to merge fields based on
which objects manage the fields instead of overruling based on values.
or
Server Side Apply tries to merge fields based on
ownership of the fields instead of overruling fields based on values.

Reply:
Thanks for looking at this, I'm not even sure I fully understand what's meant here :-)

We've spent some time finding the right words for these concepts. In general, we consider fields to be "managed" by "managers". To avoid the repetition, "actors" is also a good option. We used "owned/owners" in the past (as well as workflows but not so much) but we've tried to remove occurrences of these from the code-base.

> Server Side Apply tries to merge fields based on the actor who manages them instead of overruling just based on values.

Reply:
Thanks for reviewing!